issue 3; gitter info

jvanasco opened the pull request:
* added text about Gitter meetings
* reorganized the underlying ul/li tags regarding develop information.

zzzeek commented on develop.html.mako:
```diff
@@ -27,22 +27,40 @@ on getting started.
 <h2>Development Community</h2>
 
-<p>Where do you find the developers ?   Three places:</p>
+<p>Where do you find the developers ?  Four places:</p>
```
haha

CaselIT replied:
These is actually also https://gitter.im/sqlalchemy/community, so I guess 5 places :P

CaselIT replied:
Maybe it would be better to divide in 3 sections:
- mailing list where both are mentioned
- gitter where both channels are mentioned
- irc where again both channells are mentioned

jvanasco replied:
This is for the [/develop.html](https://www.sqlalchemy.org/develop.html) page which is aimed at code contribution.  

There is also the [/support.html](https://www.sqlalchemy.org/support.html) page, which should contain the community room (I'll add it!)

The point on this page is to showcase the developer stuff, and politely shift people to the public community stuff for everything but library/ecosystem maintenance.

CaselIT replied:
Oh ok. So maybe the other gitter link should go in the support page?

jvanasco replied:
I do like the organizational idea though, i'm going to try shifting things again!

zzzeek commented on develop.html.mako:
```diff
@@ -27,22 +27,40 @@ on getting started.
+        though we ask that discussion be limited to architectural and developmental 
+        issues only.
+        </li>
+    <li>Some development discussion occurs on the <code>#sqlalchemy-devel</code>
```
I think this should talk about gitter as where this discussion occurs right now.  the paragraph can also mention the IRC channel, but as "there is also a #sqlalchemy-devel irc channel, however currentrly the gitter channel is where most develpoment discussion is occurring".

zzzeek commented on develop.html.mako:
```diff
@@ -27,22 +27,40 @@ on getting started.
+        IRC-based support is available on the <a href="/support.html#irc">main 
+        IRC channel</a>.
+        </li>
+    <li>A real-time virtual meeting of core developers and contributers occurs in the
```
this is good to talk about the meeting, but there hsould be a separate paragraph that just discusses gitter as where development discussion occurs.

zzzeek commented on develop.html.mako:
```diff
@@ -27,22 +27,40 @@ on getting started.
+        "Subject Line" or "Title"  of the room. These meetings and their records are 
+        open to the public; users who are interested in contributing code, tests, 
+        documentation, or other developmental resources are encouraged to attend.
+        <strong>No support issues will be addressed here.</strong>
```
let's be optimistic to start (ha) and not include any of this second part of the paragraph for now (everything from the `<strong>` tag on forward).

jvanasco commented:
And yet another attempt:

* Added Gitter to the /support page
* Reorganized /develop into 'realtime' and mailing lists
* Added a line about the Code of Conduct to each one, as the code is aimed at Developers and not people asking for help - but hey everyone should be respecting it.

CaselIT commented on develop.html.mako:
```diff
@@ -27,24 +27,73 @@ on getting started.
+            discussion medium for SqlAlchemy contributors and hosts the 
+            <a href="#scheduled-developer-meetings">Scheduled Developer Meetings</a>.
+            Gitter based support is available via the
+            <a href="https://gitter.im/sqlalchemy/devel">sqlalchemy/community</a> Gitter
```
Type here: https://gitter.im/sqlalchemy/devel -> https://gitter.im/sqlalchemy/community

jvanasco replied:
great catch! thank you!

zzzeek commented on support.html.mako:
```diff
@@ -11,7 +11,7 @@ Support - SQLAlchemy
     <li><a href="#before">Before Requesting Support</a></li>
     <li><a href="#mailinglist">Mailing List</a></li>
-    <li><a href="#irc">IRC Channel</a></li>
+    <li><a href="#real-time">Real-time Channel</a></li>
```
nice

zzzeek commented on develop.html.mako:
```diff
@@ -27,24 +27,73 @@ on getting started.
 
+
+
+<a name="scheduled-developer-meetings"></a>
```
perfect
